bundles/org.openhab.binding.samsungtv/README.md

```diff
@@ -52,7 +52,7 @@ If your TV supports _websocketsecure_, you **MUST** use it, otherwise the `keyCo
 In order for the binding to control your TV, you will be asked to accept the remote connection (from openHAB) on your TV. You have 30 seconds to accept the connection. If you fail to accept it, then most channels will not work.
 Once you have accepted the connection, the returned token is stored in the binding, so you don't have to repeat this every time openHAB is restarted.
 
-If the connection has been refused, or you don't have your TV configured to allow remote connections, the binding will not work. If you are having problems, check the settings on your TV, sometimes a family member denies the popup (because they don't know what it is), and after that nothing will work.
+If the connection has been refused, or you don't have your TV configured to allow remote connections, the binding will not work. If you are having problems, check the settings on your TV. Sometimes a family member denies the popup (because they don't know what it is), and after that nothing will work.
```

Comment:
```suggestion
If the connection has been refused, or you don't have your TV configured to allow remote connections, the binding will not work.
If you are having problems, check the settings on your TV.
Sometimes a family member denies the popup (because they don't know what it is), and after that nothing will work.
```